spec/mysql2/client_spec.rb

```diff
@@ -135,15 +136,22 @@ def connect(*args)
 
     # You may need to adjust the lines below to match your SSL certificate paths
     ssl_client = nil
+    option_overrides = {
```

Comment:
This modification is to pass the test on the CentOS environment that does not have `/etc/mysql/*.pem` files.